paymentsheet/src/test/java/com/stripe/android/paymentsheet/paymentdatacollection/bacs/FakeBacsMandateConfirmationLauncher.kt

```diff
@@ -0,0 +1,12 @@
+package com.stripe.android.paymentsheet.paymentdatacollection.bacs
+
+import com.stripe.android.paymentsheet.PaymentSheet
+
+internal class FakeBacsMandateConfirmationLauncher : BacsMandateConfirmationLauncher {
```

Comment:
How would we test the entire flow with it? Can we create a fake launcher that immediately returns a hard-coded result, so that we can the confirm/modify/cancel results?